src/core/IconButton/style.ts

```diff
@@ -0,0 +1,107 @@
+import { css, SerializedStyles } from "@emotion/react";
+import styled from "@emotion/styled";
+import { IconButton } from "@material-ui/core";
+import { getColors, getSpacings, Props } from "../styles";
+
+export interface ExtraProps extends Props {
+  active?: boolean;
+  color?: "primary" | "secondary";
```

Comment:
The Figma documentation currently uses `Type` for this, but MUI has the `color` option, what do you all think?
cc @tihuan @happyimadesignr @Janeece

Reply:
Aha! I think this is another use case of what we talked at the SDS sync meeting just now 😄 

Essentially, `primary` and `secondary` describe sets of component props that SDS want to apply on the components. Such as:

"primary" -- applies color: "primary", size="medium"
"secondary" -- applies color: "default", size="small"

And in [this section](https://docs.google.com/document/d/1rFBfWWzmCiLM2HAsnjM0HIBxsB31xT7hCnYGaquOiUU/edit#bookmark=id.m7r0okvy4oxp), we are now brainstorming what name we should call the prop for "primary"/"secondary" for. E.g., phenotype, genotype, preset, etc..

Please add your ideas there too 🤩 🙏 

To unblock this PR, maybe we need @Janeece @happyimadesignr to help us clarify the difference between SDS "primary" and "secondary" here. 

E.g., if it's just whether the color is "primary" (blue) or "default" (gray), or does it include other component properties like size, variant, etc..

If it's just color, then maybe it's easier to update Figma to use color="primary"/"default" to describe using primary theme color and default for a gray?

If it's more than just describing color value, then we'll need a placeholder name for the prop for now, before we finalize a standard name in the doc 😆 Maybe.. `tbd="primary"/"secondary"` lol? Or something better!

Reply:
Not to rush us, but I think not having naming settled is starting to block us.  I think we should get a resolution as soon as possible.

Reply:
Yeah it's used so much in design! @j-x-han @ehoops do you have the bandwidth to help drive a decision on this 🙏 ? Thank you!!

Reply:
Based on our discussion, I'll update this PR to use **genotype** as the prop for primary/secondary. Does that sound right?

Reply:
Sounds right to me

Reply:
@ehoops @hthomas-czi Ah I'm thinking maybe it's `phenotype`, since primary/secondary here refers to presentation difference instead of functionality difference?

I remember Harley was using MenuSelect single select vs. multiple select an example for genotype/functionality difference! 

Please correct me if I'm wrong lol Thank you!

Reply:
Oh hmm I originally said `genotype` since the Figma doc used "Type" for primary/secondary and we decided to map type -> genotype, but I can see the case for `phenotype` since the differences here are visual rather than functional.

Are we okay with using `phenotype` here like Timmy suggested?

Reply:
To me the difference between primary/secondary icon buttons is functional. Primary Icon buttons behave more like a typical button (default, hover, click), where as the Secondary buttons have active/inactive stages that dictate the hover state. Primary icon buttons only hold an active state in rare instances, while a dropmenu is open. 

Admittedly I'm heavily skewed by how Aspen/IDseq use them. Would love input from @happyimadesignr @hthomas-czi on their perspectives.

Reply:
I agree the difference between primary and second is functional @Janeece. That's why we defined the `Type` prop in the first place, otherwise it would be a `Style` prop. 

I'm not the biggest fan of the phenotype/genotype distinction. It's clever, but maybe **too** clever. It seems unnecessarily confusing to me. I'd like to propose the following terms:

`Type` -> `Function`
`Style` -> `Appearance`

I like this because it literally describes how we've been using those props. Thoughts?

Reply:
ahh got it! So it is functional difference! Sorry for missing that 😆 

For the names, I'm still thinking `sdsStyle` and `sdsType` are short and sweet, and easier to type in code 🤔

Reply:
I think `Function` could potentially run into a lot of confusion/issues since a "function" means its own thing in coding 😅 And we could potentially be passing functions into the MUI components...

Using `sdsStyle`/`sdsType` seems like a straightforward change that doesn't require relearning what each property means, so I'm in favor of using them! What do the designers think?